applications/StructuralMechanicsApplication/custom_constitutive/linear_plane_stress.h

```diff
@@ -244,9 +244,15 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) LinearPlaneStress : public Co
     {
         //1.-Compute total deformation gradient
         const Matrix& F = rValues.GetDeformationGradientF();
-
         Matrix Etensor = prod(trans(F),F);
-        Etensor -= IdentityMatrix(2,2);
+
+        // for shells/membranes in case the DeformationGradient is of size 3x3
+        if (Etensor.size1() != 2 || Etensor.size2() != 2)
+            Etensor.resize(2,2,true); // preserve existing elements
```

Comment:
do not use the resize with preserving. We will remove it when we change from ublas to amatrix.

Reply:
So I copy the content manually?